drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -350,7 +350,8 @@ def full_name
   end
 
   def build_client_headshot_file(image_blob_id, current_user)
-    current_image_blob = ActiveStorage::Blob.find_by(id: image_blob_id)
+    # TODO(#7054) - accept either unsigned or signed blob ID for now, for backwards compatibility
+    current_image_blob = ActiveStorage::Blob.find_by(id: image_blob_id) || ActiveStorage::Blob.find_signed(image_blob_id)
```

Comment:
Would it make sense to check if `image_blob_id.to_s.to_i == image_blob_id.to_i` and then `find_by(id:` if true, and find signed if not?  That would prevent requiring two queries if `image_blob_id` is a digest/string.

Reply:
Yes good idea. It works with `image_blob_id.to_i.to_s`.